integration/container/exec_test.go

```diff
@@ -15,6 +16,75 @@ import (
 	"gotest.tools/skip"
 )
 
+// TestExecWithCloseStdin adds case for moby#37870 issue.
+func TestExecWithCloseStdin(t *testing.T) {
+	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.39"), "broken in earlier versions")
+	skip.If(t, testEnv.OSType == "windows", "FIXME. Probably needs to wait for container to be in running state.")
```

Comment:
is this WIP?

Reply:
I just followed the existing case. Not window env tests this in my local. 😢

Reply:
Could you remove the line? let's see whether CI works

Reply:
OK.